main.py

```diff
@@ -0,0 +1,10 @@
+import runner_class
+import run_race
+import asyncio
+
+first = runner_class.Runner("rabbit", 3, 0)
```

Comment:
Put Runners config (name, start, steps...) under 
./Config/runners_config
and create the runner instances in main